pyanaconda/modules/common/errors/__init__.py

```diff
@@ -0,0 +1,32 @@
+#
+# DBus errors.
```

Comment:
`Common DBus errors` or similar is better thought.

Reply:
Fixed.